client/me/billing-history/main.jsx

```diff
@@ -40,7 +40,9 @@ const BillingHistory = ( { translate } ) => (
 		<QueryBillingTransactions />
 		<PurchasesHeader section={ 'billing' } />
 		<BillingHistoryList />
-		{ config.isEnabled( 'upgrades/credit-cards' ) && <CreditCards /> }
+		{ config.isEnabled( 'upgrades/credit-cards' ) && (
```

Comment:
I believe this is enabled in all environments. I'm going to open an issue to remove it.